QA.md

```diff
@@ -83,6 +83,13 @@
     1. Generate a payment request of 2 TEST.
     1. Pay the Payment Request using your test wallet that has just received the TEST before and check the confirmation message.
 
+1. **Send HTR with QR Code**
+    1. Go to the Receive Screen and click on Payment Request.
+    1. Generate a payment request of 1 HTR
+    1. Take a screenshot of the QR Code and open that screenshot in another device
+    1. Use the Mobile Wallet QR code scanner to read this screenshot
+    1. Pay the Payment Request using the same wallet used to generate it
```

Comment:
This sounds very confusing to me

Since we are validating the QR Code scan (the QR Code itself is already being tested in another step), we can just change it to something like:

```suggestion
    1. Generate a payment request requesting a payment of 1 HTR from another device
    1. Use the Mobile Wallet QR code scanner to read this screenshot
    1. Pay the Payment Request, the transaction should be successful
```